pyproject.toml

```diff
@@ -23,7 +23,7 @@ dependencies = [
     "ipywidgets>=8.0.6",
     "voila>=0.4,<0.5",
     "pyyaml>=5.4.1",
-    "specutils>=1.9",
+    "specutils>=1.15",
```

Comment:
That is quite a jump. Do we need a change log for this?

Reply:
I would be happy to add it to the change log if you think so.

Reply:
Doesn't hurt?